internal/clients/project.go

```diff
@@ -61,6 +62,10 @@ func CreateProjectWithRetry(client *Client, params *project_service.ProjectServi
 	op := func() error {
 		var err error
 		res, err = client.Project.ProjectServiceCreate(params, nil)
+		// Wait for the project to be created, if an operation ID is returned.
+		if res.Payload.OperationID != "" {
```

Comment:
I think we should check if `err` is nil before trying to parse the response. Same with all the changes below

Reply:
if `err != nil`, we should return it immediately